Python/instrumentation.c

```diff
@@ -1303,7 +1303,6 @@ initialize_tools(PyCodeObject *code)
             opcode = DE_INSTRUMENT[opcode];
             assert(opcode != 0);
         }
-        assert(opcode != ENTER_EXECUTOR);
```

Comment:
Why remove this assert? It looks like if there is an ENTER_EXECUTOR the line `i += _PyOpcode_Caches[opcode];` (L1335 below) would do the wrong thing, since ENTER_EXECUTOR has 0 caches but it hides a JUMP_BACKWARD which does have a cache. Maybe you should move it up a bit so it's immediately after `int opcode = instr->op.code;` (L1297)?

Reply:
Good point! Applied!